docs/src/content/components/dropdown.mdx

```diff
@@ -288,7 +293,13 @@ import React from 'react'
 import Dropdown, { useDropdown } from '@pluralsight/ps-design-system-dropdown'
 import { CalendarIcon } from '@pluralsight/ps-design-system-icon'
 
-const DropdownWithIcon = React.forwardRef(
+interface DropdownWithIconProps extends Omit<HTMLAttributes<HTMLButtonElement>, 'onChange'> {
+  icon: React.ReactNode
+  onChange?: (e: React.MouseEvent, value: string) => void
```

Comment:
just double checking...should this be `value: ReactText`

Reply:
yep